core/authenticate/service.go

```diff
@@ -156,11 +166,43 @@ func (s Service) StartFlow(ctx context.Context, request RegistrationStartRequest
 		FinishURL: request.ReturnToURL,
 		CreatedAt: s.Now(),
 		ExpiresAt: s.Now().Add(defaultFlowExp),
+		Email:     request.Email,
 		Metadata: metadata.Metadata{
 			"callback_url": request.CallbackUrl,
 		},
 	}
 
+	if request.Method == PassKeyAuthMethod.String() {
+		newPassKeyUser := strategy.NewPassKeyUser(request.Email)
+		needRegistration := false
+		loggedInUser, err := s.userService.GetByID(ctx, request.Email)
+		if err != nil {
+			needRegistration = true
+		} else {
+			storedPasskey, passKeyExists := loggedInUser.Metadata["credentials"]
```

Comment:
instead of `credentials`, let's call it `passkey_credentials` to avoid confusion.

Reply:
credentials changed to  passkey_credentials